srcpkgs/libclang-python3/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'libclang-python3'
+pkgname=libclang-python3
+version=3.9.0
+revision=1
+wrksrc="libclang-py3-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-setuptools"
+short_desc="Python3 bindings for libclang"
+maintainer="shizonic <realtiaz@gmail.com>"
+homepage="https://github.com/Sublimelibclang-py3/libclang-py3"
```

Comment:
This page doesn't exist


This is the page referenced on PYPI as homepage

https://bitbucket.org/Anteru/python3-libclang

Reply:
Will change that asap.

Reply:
Fixed.